src/tcfv2/sourcepoint.ts

```diff
@@ -42,13 +44,6 @@ export const init = (pubData = {}): void => {
 			pubData: { ...pubData, cmpInitTimeUtc: new Date().getTime() },
 
 			events: {
-				onConsentReady: () => {
-					mark('cmp-tcfv2-got-consent');
```

Comment:
I think if we remove this mark we won't be able to send the `cmp-tcfv2-got-consent` metric to the lake!

Reply:
That's a good point! With @mxdvl we wanted to rename to `cmp-tcfv2-tc-loaded` marked [here](https://github.com/guardian/consent-management-platform/pull/464/files#diff-6bda0a51ba14b83496ad086305548ce826581089bde25cbd43c4309c505454cdR46). But that does complicate our two changes.
How about attempting to collect both `cmp-tcfv2-got-consent` and `cmp-tcfv2-tc-loaded` on your branch, then I can remove `cmp-tcfv2-got-consent` once this PR is merged?